internal/app/tfsec/block/hclblock.go

```diff
@@ -208,6 +211,27 @@ func (block *HCLBlock) GetAttribute(name string) Attribute {
 	return nil
 }
 
+func (block *HCLBlock) GetNestedAttribute(name string) Attribute {
+	parts := strings.Split(name, "/")
+	blocks := parts[:len(parts)-1]
+	attrName := parts[len(parts)-1]
+
+	working := block
```

Comment:
If we cast it to the interface we can remove the cast on line 224 and it'll be future-proofed.
```suggestion
	working := block.(Block)
```

Reply:
Done